pkgs/development/libraries/postgis/default.nix

```diff
@@ -5,7 +5,7 @@ args@{fetchurl, composableDerivation, stdenv, perl, libxml2, postgresql, geos, p
   ### NixOS - usage:
```

Comment:
From https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md  ,
commit subject should be "postgis: 2.2.1 -> 2.3.1"